pwa/src/templates/organizationDetail/OrganizationDetailTemplate.tsx

```diff
@@ -31,158 +33,174 @@ interface OrganizationDetailTemplateProps {
-  const TempComponentsOwned = TEMPORARY_COMPONENTS.slice(0, 7);
-  const TempComponentsSupported = TEMPORARY_COMPONENTS.slice(8, 13);
-  const TempComponentsUsed = TEMPORARY_COMPONENTS;
-
-  const tempOrganization =
-    TEMPORARY_ORGANIZATIONS.find((organization) => organization.id === organizationId) ?? TEMPORARY_ORGANIZATIONS[1];
+  const queryClient = new QueryClient();
+  const _useOrganization = useOrganization(queryClient);
+  const _getOrganization = _useOrganization.getOne(organizationId);
 
   return (
     <Container layoutClassName={styles.container}>
-      <div className={styles.headerContainer}>
-        <div className={styles.headerContent}>
-          <Heading1>{tempOrganization.name}</Heading1>
-
-          <LeadParagraph>{tempOrganization.description}</LeadParagraph>
-        </div>
+      {_getOrganization.isSuccess && (
+        <>
+          <div className={styles.headerContainer}>
+            <div className={styles.headerContent}>
+              <Heading1>{_getOrganization.data.name}</Heading1>
 
-        <div className={styles.headerOrganizationData}>
-          <div className={styles.logoContainer}>
-            <img className={styles.logo} src={tempOrganization.logo} alt="Organization logo" />
-          </div>
-          <div>
-            <div className={styles.tagsContainer}>
-              {tempOrganization.github && (
-                <ToolTip tooltip="GitHub">
-                  <Tag label={t("GitHub")} icon={<GitHubLogo />} onClick={() => open(tempOrganization.github)} />
-                </ToolTip>
-              )}
-              {tempOrganization.gitlab && (
-                <ToolTip tooltip="GitLab">
-                  <Tag label={t("GitLab")} icon={<GitLabLogo />} onClick={() => open(tempOrganization.gitlab)} />
-                </ToolTip>
+              {!_getOrganization.data.description && <LeadParagraph>{_getOrganization.data.description}</LeadParagraph>}
+              {!_getOrganization.data.description && (
+                <LeadParagraph>{t("There is no description available")}</LeadParagraph>
               )}
-              {tempOrganization.website && (
-                <ToolTip tooltip={"Website"}>
-                  <Tag
-                    label={tempOrganization.website}
-                    icon={<FontAwesomeIcon icon={faGlobe} />}
-                    onClick={() => open(tempOrganization.website)}
-                  />
-                </ToolTip>
-              )}
-              {tempOrganization.phone && (
-                <ToolTip tooltip={"Telefoonnummer"}>
+            </div>
+
+            <div className={styles.headerOrganizationData}>
+              <div className={styles.logoContainer}>
+                {_getOrganization.data.logo && (
+                  <img className={styles.logo} src={_getOrganization.data.logo} alt="Organization logo" />
+                )}
+                {!_getOrganization.data.logo && <img src={grey} className={styles.logo} />}
```

Comment:
Could this be simplified by doing: `<img src={_getOrganization.data.logo ?? organizationPlaceholderImage} />`?